services/transcode/audio.go

```diff
@@ -41,5 +44,73 @@ func AudioAac(input common.AudioInput, cb ffmpeg.ProgressCallback) (*common.Audi
 
 	return &common.AudioResult{
 		OutputPath: outputPath,
+		Bitrate:    input.Bitrate,
+		Format:     "aac",
+	}, nil
+}
+
+func getQfactorFromBitrate(input string) int {
+
+	bitrate, _ := strconv.ParseInt(strings.ReplaceAll(input, "k", ""), 10, 64)
+
+	switch {
+	case bitrate >= 190:
+		return 1
+	case bitrate >= 170:
+		return 2
+	case bitrate >= 150:
+		return 3
+	case bitrate >= 140:
+		return 4
+	case bitrate >= 120:
+		return 5
+	case bitrate >= 100:
+		return 6
+	case bitrate >= 80:
+		return 7
+	case bitrate >= 70:
+		return 8
+	case bitrate >= 45:
+		return 9
+	default:
+		return 1
+	}
+}
+
+func AudioMP3(input common.AudioInput, cb ffmpeg.ProgressCallback) (*common.AudioResult, error) {
```

Comment:
I think it would be cool to have simple unit tests for all the ffmpeg `params`. I did dumb golden tests for xdcam and playout but I think it can be helpful

Reply:
I have not added tests here yet, because I think we need to change a bit how this works. Will do a refactor in a different PR, together with the DIR/Folder stuff.